verbose_debugging_reports.md

```diff
@@ -0,0 +1,413 @@
+Verbose Debugging Reports
+=========================
+
+This document is a collection of [verbose debugging
+reports](https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#optional-verbose-debugging-reports)
+that are supported.
+
+### Source debugging reports
+
+Here are the debugging reports supported for [attribution source
+registrations](https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#registering-attribution-sources).
+
+#### Destination limit reached
+
+* Reason
+  * a source is rejected due to the [destination limit](https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#limiting-the-number-of-unique-destinations-covered-by-unexpired-sources)
+* Report
+
+```jsonc
+{
+  "type": "source-destination-limit",
+  "body": {
+    "attribution_destination": "https://destination.example",
+    "limit": "100", // the browser's limit
+    "source_debug_key": "<debug key in the source registration>", // omitted if not set
+    "source_event_id": "<source event id in the source registration>",
+    "source_site": "https://source.example"
+  }
+}
```

Comment:
Instead of repeating this with minor variations for each report type, what do you think about a table like:

| `type` | `attribution_destination` | `limit` | `source_debug_key` | `source_event_id` | `source_site` | `trigger_debug_key` |
| --- | --- | --- |--- |--- |--- | --- |
| `source-destination-limit` | ✓ | ✓ | ✓ | ✓ | ✓ |
| `source-noised` | ✓ | ❌ | ✓ | ✓ | ✓ |
| `source-storage-limit` | ✓ | ✓ | ✓ | ✓ | ✓ |
| `source-unknown-error` | ✓ | ❌ | ✓ | ✓ | ✓ |
| `trigger-no-matching-source` | ✓ | ❌ | ❌ | ❌ | ❌ | ✓ |
| `trigger-no-matching-filter-data` | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
| `trigger-attributions-per-source-destination-limit` | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
| `trigger-reporting-origin-limit` | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
| `trigger-event-deduplicated` | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
| `trigger-event-no-matching-configurations` | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
| `trigger-event-noise` | ✓ | ❌ | ✓ | ✓ | ✓ | ✓ |
| `trigger-event-storage-limit` | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |
| `trigger-event-report-window-passed` | ✓ | ✓ | ✓ | ✓ | ✓ | ✓ |

This could be followed by a description and example of each of the fields, e.g.

<dl>
<dt><code>attribution_destination</code></dt>
<dd>The site on which attribution did/would occur, e.g. <code>"https://destination.example"</code>.
</dl>

Reply:
Thanks Andrew. I thought about having a table, but I feel repeating the fields is slightly clearer and easier to read. I don't feel strongly though, wdyt?

Reply:
Gave it a try with the table, PTAL, thanks!